typescript/src/events/guilds/messages/guildUserMessageImageNotify.ts

```diff
@@ -27,13 +26,14 @@ export class UserEvent extends Event {
 		// If the message was edited, do not repost:
 		if (message.editedTimestamp) return;
 
-		const [logChannel, ignoredChannels] = await message.guild.readSettings([
-			GuildSettings.Channels.Logs.Image,
-			GuildSettings.Channels.Ignore.All
+		const key = GuildSettings.Channels.Logs.Image;
+		const [channelId, ignoredChannels, t] = await message.guild.readSettings((settings) => [
```

Comment:
```suggestion
		const [logChannelId, ignoredChannels, t] = await message.guild.readSettings((settings) => [
```
And rename references accordingly. I had to check what the heck `channelId` was, a simple other name makes it easier already.